www/src/components/utils/Header.js

```diff
@@ -0,0 +1,26 @@
+import { Box } from 'grommet'
+import { Flex, H1, Span } from 'honorable'
+
+export function Header({ header, description, children }) {
```

Comment:
Is it used? I didn't find any usage in the new code.

Reply:
It's imported in `plural/www/src/components/account/AccountAttributes.js` and `plural/www/src/components/repository/OIDCProvider.js`.

Reply:
It's imported from `src/components/profile/Header.js` there, not this one from what I can see.

Reply:
https://github.com/pluralsh/plural/blob/9b66cc61305f31398bb324871bc68c5acdd72cd0/www/src/components/account/AccountAttributes.js#L10-L11

Reply:
You're right, good catch. I've started this to move Header to utils as it's used not only in the profile but across whole app. Probably switched to something and forgot to finish.